build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -55,6 +55,21 @@ spec:
                   minimum: 1
                   maximum: 65535
                   default: 179
+                confederation:
+                  type: object
+                  properties:
+                    identifier:
+                      type: integer
+                      format: int32
+                      minimum: 1
+                      maximum: 65535
+                    peers:
+                      type: array
+                      items:
+                        type: integer
+                        format: int32
+                        minimum: 1
+                        maximum: 65535
```

Comment:
I saw there is a suggestion is to use private ASN 64512–65534 for peers. But I am not sure if it's mandatory or just a recommendation. Could you double check the range? 
If it's mandatory, we may need to remind users in the guide later when the full feature is ready. Thanks.

Reply:
Is this one https://github.com/antrea-io/antrea/pull/6905#discussion_r1919317524?

Reply:
I think private ASNs are typically used inside a confederation but I don't believe it's required. We can mention it in the documentation as you suggested.

Reply:
Will add.